tests/e2e/json_utils.go

```diff
@@ -0,0 +1,412 @@
+		var a voteGovProposalAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.startConsumerChainAction":
+		var a startConsumerChainAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.addChainToRelayerAction":
+		var a addChainToRelayerAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.addIbcConnectionAction":
+		var a addIbcConnectionAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.addIbcChannelAction":
+		var a addIbcChannelAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.transferChannelCompleteAction":
+		var a transferChannelCompleteAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.unjailValidatorAction":
+		var a unjailValidatorAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.assignConsumerPubKeyAction":
+		var a assignConsumerPubKeyAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.delegateTokensAction":
+		var a delegateTokensAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.relayPacketsAction":
+		var a relayPacketsAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.registerRepresentativeAction":
+		var a registerRepresentativeAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.relayRewardPacketsToProviderAction":
+		var a relayRewardPacketsToProviderAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.submitChangeRewardDenomsProposalAction":
+		var a submitChangeRewardDenomsProposalAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.downtimeSlashAction":
+		var a downtimeSlashAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.unbondTokensAction":
+		var a unbondTokensAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.cancelUnbondTokensAction":
+		var a cancelUnbondTokensAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.redelegateTokensAction":
+		var a redelegateTokensAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.doublesignSlashAction":
+		var a doublesignSlashAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.startRelayerAction":
+		var a startRelayerAction
+		err := json.Unmarshal(rawAction, &a)
+		if err == nil {
+			return a, nil
+		}
+	case "main.slashThrottleDequeue":
+		var a slashThrottleDequeue
```

Comment:
Can we refactor `slashThrottleDequeue` to be `slashThrottleDequeueAction` as well?